src/asset_folder_importer/asset_folder_vsingester/importer_thread.py

```diff
@@ -416,12 +424,14 @@ def attempt_file_import(self, fileref, filepath, rootpath):
                         logging.warning("Received {0} HTTP error from Vidispine. Retrying.".format(e.code))
 
                 attempts += 1
+
+        project_from_pluto = self.ask_pluto_for_projectid(os.path.join(fileref['filepath'], fileref['filename']))
         
```

Comment:
minor thing, an INFO log line would be nice here to say "found project id {projectid} for item at {filepath}"

Reply:
Done.